### PR TITLE
EnvironmentNameNotGettingReflected

### PR DESCRIPTION
**Pull request by rathimayur**
When Env name getting modified it doesn't reflect in Grid Title
Same issue with Data Source page also, fixed in both places.

Thank you for your contribution.
Before submitting this PR, please make sure:
- [ ] PR description and commit message should describe the changes done in this PR
- [ ] Verify the PR is pointing to correct branch i.e. Release or Beta branch if the code fix is for specific release , else point it to master
- [ ] Latest Code from master or specific release branch is merged to your branch
- [ ] No unwanted\commented\junk code is included
- [ ] No new warning upon build solution
- [ ] Code Summary\Comments are added to my code which explains what my code is doing
- [ ] Existing unit test cases are passed
- [ ] New Unit tests are added for your development
- [ ] Sanity Tests are successfully executed for New and Existing Functionality
- [ ] Verify that changes are compatible with all relevant browsers and platforms.
- [ ] After creating pull request there should not be any conflicts
- [ ] Resolve codacy comments
- [ ] Builds and checks are passed before PR is sent for review
- [ ] Resolve code review comments
- [ ] Help Libray document is updated accordingly for Feature changes.

### CODE REVIEW

File: Ginger/Ginger/DataSource/DataSourcePage.xaml.cs [hunk 1]
```diff
@@ -202,5 +202,10 @@ private void AddTable(object sender, RoutedEventArgs e)
         }
 
         #endregion Functions
+
+        private void DataSourceNameTextBox_TextChanged(object sender, TextChangedEventArgs e)
+        {
+            grdTableList.Title = "'" + mDSDetails.Name + "' Tables List";
```
Comment (Maheshkale447) [inline Ginger/Ginger/DataSource/DataSourcePage.xaml.cs:208]: Please use String interpolation

File: Ginger/Ginger/Environments/AppsListPage.xaml.cs [hunk 2]
```diff
@@ -142,5 +142,10 @@ private void AddAppsToOtherEnvironments(object sender, RoutedEventArgs e)
         }
 
         #endregion Functions
+
+        private void EnvNameTextBox_TextChanged(object sender, TextChangedEventArgs e)
+        {
+            grdApps.Title = "'" + EnvNameTextBox.Text + "' Environment Applications";
```
Comment (Maheshkale447) [inline Ginger/Ginger/Environments/AppsListPage.xaml.cs:148]: Please use String interpolation instead of concatenating string with +

**Review by Maheshkale447 — CHANGES_REQUESTED** [review]

File: Ginger/Ginger/Environments/AppsListPage.xaml.cs (hunk 2 above)
Comment (rathimayur) [reply Ginger/Ginger/Environments/AppsListPage.xaml.cs]: Handled.

File: Ginger/Ginger/DataSource/DataSourcePage.xaml.cs [hunk 3]
```diff
@@ -65,7 +65,7 @@ public DataSourcePage(DataSourceBase dsDetails)
         private void SetGridView()
         {
             //Set the grid name
-            grdTableList.Title = "'" + mDSDetails.Name + "' Tables List";
+            grdTableList.Title = $"'{mDSDetails.Name} \"' Tables List";
```
Comment (Maheshkale447) [inline Ginger/Ginger/DataSource/DataSourcePage.xaml.cs:68]: Is \" needed in string?

**Review by Maheshkale447 — CHANGES_REQUESTED** [review]

File: Ginger/Ginger/DataSource/DataSourcePage.xaml.cs (hunk 3 above)
Comment (rathimayur) [reply Ginger/Ginger/DataSource/DataSourcePage.xaml.cs]: Removed unnecessary " from string.

**Review by Maheshkale447 — APPROVED** [review]

**Pull request merged**